docs/source/hyperparameters.rst

```diff
@@ -136,14 +137,32 @@ Now pass in the params when you init your model
 
     parser = ArgumentParser()
     parser = LitMNIST.add_model_specific_args(parser)
-    hparams = parser.parse_args()
-    model = LitMNIST(hparams)
+    args = parser.parse_args()
+    model = LitMNIST(args)
```

Comment:
Should raise an exception, as LitMNIST expects multiple arguments, not a Namespace